Local Pods/GitHubAPI/GitHubAPI/GitHubAPIStatusRequest.swift

```diff
@@ -9,15 +9,22 @@
 import Foundation
 
 public struct APIStatus: Codable {
-    public enum StatusType: String, Codable {
-        case good, minor, major
+    public let status: Status
+}
+
+public struct Status: Codable {
+    public let indicator: StatusType
+    public let description: String
+    
+    public enum StatusType: String,Codable, CodingKey {
```

Comment:
Can we rename the `case none` to `case normal` or similar? `.none` can silently be interpreted as `Optional.none` when working with an optional enum of this type, which can be hard to debug. Would require manually handling the decoding though.

Reply:
Decided against changing this for now as it would also "change" the API, which might be confusing, and certainly would be more confusing in most cases.